css/index.css

```diff
@@ -112,6 +151,48 @@ p {
     justify-content: center;
 }
 
```

Comment:
Excellent!  You used parent/child relationships and let the parent .box style all the individual boxes except where the color of each box needed to be changed.